### PR TITLE
*: Add contributing guide, DCO, CoC

### PR DESCRIPTION
**Pull request by ajeddeloh**
Add contributing guide, developer certificate of origin and code of
conduct.

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+principles to every aspect of managing this project. Project maintainers who do
+not follow or enforce the Code of Conduct may be permanently removed from the
+project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting a project maintainer or Brandon Philips
+<brandon.philips@coreos.com>.
```
Comment (arithx) [inline code-of-conduct.md]: Should we have a different contact point?

Comment (ashcrow) [inline code-of-conduct.md]: Is this still accurate?

**Review by arithx — APPROVED** [review]
LGTM, one question

File: code-of-conduct.md (hunk 1 above)
Comment (ajeddeloh) [reply code-of-conduct.md]: cc @philips Good question

Comment (philips) [reply code-of-conduct.md]: Yes, I don't know what it should be. Probably a list of maintainers at this point. Ideally someone who has had some sort of Code of Conduct response training or at the very least read a book on the topic like this: https://frameshiftconsulting.com/code-of-conduct-book/

Comment (ashcrow) [reply code-of-conduct.md]: That makes sense. Thanks @philips. 

@ajeddeloh @arithx Why don't we leave it at `contacting a project maintainer from MAINTAINERS` for now.

Comment (ajeddeloh) [reply code-of-conduct.md]: SGTM. Added a MAINTAINERS file and dropped brandon.

**Review by arithx — APPROVED** [review]
LGTM

**Pull request merged**